tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -113,6 +113,52 @@ Search and Verify GPU Items Appears In Resources Page
     Search Items In Resources Section   GPU
     Check GPU Resources
 
+Verify Advanced filtering by status, provider, types from Resources tab is working as expected
```

Comment:
Test case title should be in TitleCase

Main tests should be simplified in simpler keywords. See the style guide for test complexity examples